core/src/main/java/org/infinispan/container/offheap/BoundedOffHeapDataContainer.java

```diff
@@ -73,103 +73,106 @@ protected void entryReplaced(long newAddress, long oldAddress) {
       long newSize = sizeCalculator.applyAsLong(newAddress);
       lruLock.lock();
       try {
-         long lruNode = offHeapEntryFactory.getLruNode(oldAddress);
          if (trace) {
-            log.tracef("Replacing LRU node: 0x%016x. OldValue: 0x%016x NewValue: 0x%016x", lruNode, oldAddress,
+            log.tracef("Replacing LRU node: OldValue: 0x%016x NewValue: 0x%016x", oldAddress,
```

Comment:
`removeNode` and `addEntryAddressToEnd` both log, so maybe we can remove this log ,essage.

Reply:
Okay